sections/main-list-collections.liquid

```diff
@@ -1,21 +1,15 @@
 {{ 'queries-handler.css' | asset_url | stylesheet_tag }}
-{{ 'featured-products.css' | asset_url | stylesheet_tag }}
+{{ 'collection-listing.css' | asset_url | stylesheet_tag }}
 
 {%- assign per_page = section.settings.items_per_page -%}
-<div class='yc-featured-products container'>
```

Comment:
<img width="295" alt="Screen Shot 2023-02-21 at 17 24 30" src="https://user-images.githubusercontent.com/93322743/220402332-b1d221e4-eef2-486d-b56d-b367c5fe802b.png">

you need to remove title settings also